src/components/Postiterie.vue

```diff
@@ -146,13 +151,20 @@ export default {
     },
 
 
-
     // function resultToXML
     // TODO
     // return the XML from the result array[];
     resultToXML: function(){
-      let xml = "";
-    
+      let $this = this;
+      let xml = '<?xml version="1.0" encoding="UTF-8"?><mxGraphModel dx="1190" dy="757" grid="1" gridSize="10" guides="1" tooltips="1" connect="1" arrows="1" fold="1" page="1" pageScale="1" pageWidth="827" pageHeight="1169" math="0" shadow="0"><root><mxCell id="0" /><mxCell id="1" parent="0" />';
+      this.temps.forEach(function (el, index) {
+        let base = el.replace(";base64", "");
+        xml+=`<mxCell id="xVlBypw9ISABlRlfdyYR-`+index+`" value="" 
```

Comment:
This needs to be elsewhere. Having this kind of huge string within the code is very awkward. Hide it in another file and import it?

Reply:
Also, use string templating : xml = `<blablabla>${anotherVariable}</blablabla>`

Reply:
put xml variable in .env and the repetitive string in a separate function